internal/cmd/envs.go

```diff
@@ -62,8 +62,115 @@ func envsCommand() *cobra.Command {
 	}
 	lsCmd.Flags().StringVarP(&outputFmt, "output", "o", "human", "human | json")
 	cmd.AddCommand(lsCmd)
-	cmd.AddCommand(stopEnvCommand(&user))
 
+	var (
+		img      string
+		tag      string
+		cpuCores float32
+		memGB    float32
+		diskGB   int
+		gpus     int
+		services []string
+	)
+
+	editEnvCmd := &cobra.Command{
+		Use:    "edit",
+		Short:  "edit an existing environment owned by the active user.",
+		Args:   cobra.ExactArgs(1),
+		Hidden: true,
+		Long:   "Edit an existing environment owned by the active user.",
+		Example: `coder envs edit back-end-env
+# increase number of cores for back-end env
```

Comment:
not necessarily increasing. this one seems pretty self-explanatory such that we can remove the comment.

Reply:
Also, we should probably error if they call `coder envs edit [env]` and suggest that they use `coder envs rebuild [env]` instead.